dev_guide/integrating_external_services.adoc

```diff
@@ -133,9 +134,13 @@ Use an `ExternalName` service without defining any ports or endpoints.
     externalName: example.domain.name
   selector: {} <1>
 ----
-
 <1> The `selector` field to leave blank.
 
+Using an `ExternalName` service maps the service to the value of the
```

Comment:
How about:
Using an `ExternalName` service tells the system that the DNS name in `externalName` field (`example.domain.name` in the above example) is where the resource that backs the service is.  When a DNS request is made against the Kubernetes DNS server, it will return the `externalName` in a CNAME record telling the client to look up the returned name to get the IP address.  This makes it easier to manage an external service linkage because you don't have to worry about the external service's IP addresses changing.